### ISSUE TITLE
App occasionally records approximately 10x actual steps for a day

### ISSUE THREAD

**Issue by ZornsLemma** [j4velin/Pedometer#121]
I've been using the app for a couple of months now, and generally it works great, thank you! However, in my first week of using it and again just yesterday I had one day show up with something like 200k (not 2 million, as in one of the closed issues) steps, when I probably did about 10k. This is on a Blackberry Priv, for what it's worth. I don't have any reliable way to reproduce this, unfortunately. I didn't to the best of my knowledge do any unusual activities on the two days when this happened.

It would be nice if I could at least discard these days from my statistics, either automatically or via some kind of manual process. Perhaps there could be a user-settable cap and the number of steps per day is never allowed to be recorded as more than that? Or (more difficult, I'm sure) perhaps the average number of steps per second between readings of the hardware step counter could be calculated, and if it shows a rate above some value the reading is discarded. (I don't know how often the hardware step counter is read; if it's read every half hour or so that might work, if it's once per day it's obviously not going to work.)

If there's anything I can do to help debug this problem please let me know, and thanks very much for making the app available in the first place!

**Comment by jerryasher** [comment]
Yes, I've seen this behavior too. It's not that it's misreading my actual steps, it's that for whatever reason it will register thousands (as in 30 thousands) of steps in a very short span when the phone is actually stationary.

I would like to be able to

+ discard a day
+ edit a day's steps
